### PR TITLE
Employee Management Page - Data Retrieval

### PR DESCRIPTION
**Pull request by osaosula**
Employee data retrieval from endpoint

### CODE REVIEW

File: src/pages/Employees.jsx [hunk 1]
```diff
@@ -1,77 +1,18 @@
-import { useState } from "react";
 import { AgGridReact } from "ag-grid-react";
 import "ag-grid-community/styles/ag-grid.css";
 import "ag-grid-community/styles/ag-theme-alpine.css";
-
+import { getEployees } from "../services/apiEmployees";
```
Comment (dellydel) [inline src/pages/Employees.jsx:4]: Small spelling error

File: src/pages/Employees.jsx [hunk 2]
```diff
@@ -1,77 +1,18 @@
...
-      lastLogin: "25-Jul-202414:48:24",
-    },
-  ]);
+  const {
+    isPending,
+    data: employees,
+    error,
+  } = useQuery({
+    queryKey: ["employees"],
+    queryFn: getEployees,
```
Comment (dellydel) [inline src/pages/Employees.jsx:13]: Spelling

File: src/services/apiEmployees.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+export async function getEployees() {
```
Comment (dellydel) [inline src/services/apiEmployees.js:2]: Need to use URL from environment variable so that it will be generated at build time depending on dev or prod

**Review by dellydel — CHANGES_REQUESTED** [review]
Few small changes

File: src/services/apiEmployees.js (hunk 3 above)
Comment (osaosula) [reply src/services/apiEmployees.js:2]: Corrected

File: src/pages/Employees.jsx (hunk 1 above)
Comment (osaosula) [reply src/pages/Employees.jsx:4]: Fixed

File: src/pages/Employees.jsx (hunk 2 above)
Comment (osaosula) [reply src/pages/Employees.jsx:13]: Fixed

**Review by dellydel — APPROVED** [review]

**Review by BorisLaboratory — APPROVED** [review]
Looks good

**Pull request merged**